test/e2e/framework/workspaces.go

```diff
@@ -106,7 +106,7 @@ func WithName(s string, formatArgs ...interface{}) UnprivilegedWorkspaceOption {
 	}
 }
 
-func NewWorkspaceFixtureObject[O WorkspaceOption](t *testing.T, clusterClient kcpclientset.ClusterInterface, parent logicalcluster.Path, options ...O) *tenancyv1beta1.Workspace {
+func newWorkspaceFixture[O WorkspaceOption](t *testing.T, clusterClient kcpclientset.ClusterInterface, parent logicalcluster.Path, options ...O) *tenancyv1beta1.Workspace {
```

Comment:
Can we emit some logs inside of this to be explicit about the `logicalcluster.Path` -> `logicalcluster.Name` mapping? So that you can do the translation post-hoc

Reply:
good idea

Reply:
done